lxqt-admin-time/translations/lxqt-admin-time_pt.ts

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.0" language="pt">
+<context>
+    <name>TimeAdminDialog</name>
+    <message>
+        <location filename="../timeadmindialog.ui" line="14"/>
+        <source>Time and Date Settings</source>
+        <translation>Definições de data e hora</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="23"/>
+        <source>Set current date and time</source>
+        <translation>Definir data e hora atual</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="30"/>
+        <source>Time zone:</source>
+        <translation>Fuso horário:</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="40"/>
+        <source>Time:</source>
+        <translation>Hora:</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="47"/>
+        <source>hh:mm:ss AP</source>
```

Comment:
@smarquespt The source file you used isn't the latest one. This file is from commit 33191591c9f0b5164bb007bdc68a18f87f3427f4
Please use the latest .ts file.
This can't be merged.